### PR TITLE
add admin words index page

### PR DESCRIPTION
**Pull request by kian1213**
Hello sir, please check my pull request. Thanks

### CODE REVIEW

**Comment by codegracie** [comment]
Okay.

File: app/controllers/admin/words_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Admin::WordsController < AdminController
+  def index
+    @words = Word.paginate :page => params[:page], :per_page => 10
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: `@words = Word.paginate page: params[:page], per_page: 10`

File: app/controllers/admin/words_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def show
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/controllers/admin/words_controller.rb (hunk 1 above)
Comment (SSalekin) [inline app/controllers/admin/words_controller.rb]: convention pls.

File: app/controllers/admin/words_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def new
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/controllers/admin/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def create
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/controllers/admin/words_controller.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def edit
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/controllers/admin/words_controller.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def update
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/controllers/admin/words_controller.rb [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def destroy
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: this pull only `add admin words index page`! So clear it

File: app/views/admin/words/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+              </td>
+              <td>
+                <div class="admin-actions">
+                  <%= link_to admin_word_path(word), class: "button btn btn-success btn-xs" do %>
+                    <i class="fa fa-file-text"></i>
+                  <% end %>
+                  <%= link_to edit_admin_word_path(word), class: "button btn btn-primary btn-xs" do %>
+                    <i class="fa fa-pencil"></i>
+                  <% end %>
+                  <%= link_to admin_word_path(word), method: :delete, data: {confirm: "You want to delete?"}, class: "button btn btn-danger btn-xs" do %>
```
Comment (SSalekin) [inline app/views/admin/words/index.html.erb]: 80 char rule.

**Pull request merged**